plugins/Diagnostics/Diagnostic/AbstractPrivateDirectories.php

```diff
@@ -65,6 +66,18 @@ public function execute()
         }
 
         $result = new DiagnosticResult($label);
+        if (Config::getInstance()->General['enable_required_directories_diagnostic'] == 0) {
+            if (!$this->addedDisableHttpDiagnosticsWarning) {
```

Comment:
@geekdenz the `addedDisableHttpDiagnosticsWarning` property was removed but here it's still in the code

Reply:
Oh man! Sorry! Stupid mistake. Thought it was a trivial change, did it in vim quickly and was planning to wait for the ci tests and forgot to put it in draft.